src/Interpreters/FillingRow.cpp

```diff
@@ -49,7 +49,7 @@ bool FillingRow::next(const FillingRow & to_row)
     size_t pos = 0;
 
     /// Find position we need to increment for generating next row.
-    for (; pos < size(); ++pos)
```

Comment:
This is a minor change, but the compiler cannot typically optimize size calls out of the loop due to not enough strict-aliasing rules in C++ (in comparison to Rust).